vagrant-pxe-harvester/ansible/setup_harvester.yml

```diff
@@ -50,12 +40,23 @@
       node_number: "{{ item }}"
     with_sequence: 0-{{ harvester_cluster_nodes|int - 1 }}
 
-  - name: create "Installation Completed" message
+  - name: install Rancher {{ harvester_network_config.rancher.version }}
```

Comment:
nit: we can put the "rancher" setting to the top level. The field doesn't look appropriate to live under "harvester_network_config".

Reply:
Agree, fixed in commit [3be3f29](https://github.com/harvester/ipxe-examples/pull/66/commits/3be3f29e81f35a46468358e598806c40d84df00e).